jenkins/release-tag/release-tag.jenkinsfile

```diff
@@ -0,0 +1,54 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+
+pipeline {
+    agent {
+        docker {
+            label 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1-20211130'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'BUILD_ID',
+            description: 'Build ID of the distribution build manifest generated.',
+            trim: true
+        )
+        string(
+            name: 'VERSION',
+            description: 'Version of the distribution build manifest.',
+            trim: true
+        )
+    }
+    environment {
+            BUILD_MANIFEST = "builds/opensearch/manifest.yml"
```

Comment:
Can this variable be local to the stage? Let's avoid global variables as much as possible

Reply:
Actually no. I initially had this in local but moved this to env variable for testing purpose.

Reply:
Is it used anywhere outside of the job?

Reply:
No. This is only used as the place we store the manifest downloaded and used for fetching the info we needed. While I have this as a fixed location in this job, I used a different manifest path from our testing directory. (`tests/data/opensearch-build-1.1.0.yml`). If I defined this variable in the local, we can't perform testing without actually downloading the manifest to here.

Reply:
I think overriding the variable using binding should work. Let's try this offline

Reply:
No it won't. That's what I tried initially and why I switched to what it's now.